### DISCUSSION TITLE
New article in Annals of Glaciology

### DISCUSSION THREAD

**Discussion by eberthie** [IACS-cryo/Delineation-WG#17] [Paper discussions]
[Updating glacier inventories on the periphery of Antarctica and Greenland using multi-source data.
](https://www.cambridge.org/core/journals/annals-of-glaciology/article/updating-glacier-inventories-on-the-periphery-of-antarctica-and-greenland-using-multisource-data/BBEAF2BE216E20E535066C39B6E6ED78
). The title is maybe a bit misleading: these are not complete updated inventories but restricted to specific test areas. Still the authors could be contacted (if they are not part of the group).

**Comment by fmaussion** [comment]
Thanks Etienne! I've seen that as well (title was misleading, I needed quite some time to find out that its only test areas)

**Reply by mankoff** [reply]
I'm on limited internet and not downloading a 32 MB PDF for a few more weeks, but given the abstract it seems like we should invite these authors to be a part of this effort.

**Reply by eberthie** [reply]
Here is the contact of the Corresponding author: Lu An; Email: Anlu2021@tongji.edu.cn

**Reply by MathieuMorlighem** [reply]
Hey I know An Lu very well! She was a PhD student at UCI when I was there (Eric Rignot was her advisor). Happy to reach out if ou would like.
